### DISCUSSION TITLE
Write speed too slow

### DISCUSSION THREAD

**Discussion by resercher21** [Chia-Network/chia-blockchain#4568] [Support]
When I was on Disk P, the first k32 data was written faster, about half an hour. The next five k32 data were written for seven hours. What's the reason?
My hard disk is a RAID5 array with USB 3.0 interface.

**Comment by lvcivs** [comment]
Are you using HDD or SSD?

SSDs are known to perform worse when they are almost full.

**Reply by resercher21** [reply]
plot SSD、mining HHD

**Reply by resercher21** [reply]
intel P3700, and I have 200 GB of free data space

**Reply by lvcivs** [reply]
Make sure that  you don't have any left over temp data from previous plotting in your temp directory. Sometimes the plotting gets interrupted and you have to manually clean up the temp files. This could fill up your disk and slow it down.

Also, what is the reason for the RAID5? Unless they are super tiny drives I don't really see a reason for it.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
